### DISCUSSION TITLE
How to change the position of Hamburger menu icon - .Net 8 Maui

### DISCUSSION THREAD

**Discussion by VaibhavMethuku01** [dotnet/maui#21862] [Q&A]
### Description

recently upgraded my MAUI project from .Net7 to .Net8 and i"m facing some issues with the hamburger icon placement which is distorted after the upgradation

We have tried adding a renderer which is responsible to set the content off on the Nav bar. Here is the code snippet

`WindowHandler.Mapper.AppendToMapping(nameof(IWindow), (handler, view) => { if (view.Handler != null) ((MauiWinUIWindow)view.Handler.PlatformView)!.ExtendsContentIntoTitleBar = false; });`

Expected behavior
Position of hamburger menu icon should be below the bar
![image](https://github.com/dotnet/maui/assets/155944444/c94cc635-414e-41d0-986d-b5df5f2a550d)

Actual behavior
But in Net8 it's not working and this is how it looks, the Hamburger menu is on the Nav bar
![image](https://github.com/dotnet/maui/assets/155944444/74dec53b-4acd-4179-b36e-d6c41b287a97)

### Steps to Reproduce

Create a .Net 8 Maui project and observe the hamburger menu position.

### Version with bug

8.0.20 SR4

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

7.0.100

### Affected platforms

Windows

### Affected platform versions

Windows

**Comment by Xynratron** [comment]
I currently migrate from Xamarin to MAUI and face the same issue.

Is there any workaround? Because .NET 7 is out of support, so I must use .NET 8.

**Comment by Xynratron** [comment]
I just play a bit around with the Maui Community Toolkit. There is a solution for the Problem:

![image](https://github.com/user-attachments/assets/7c776ce1-618e-4008-842a-ff7f8ccd08ec)

Have a look at the CommunityToolkit.Maui.Sample in MauiProgram.cs there is a snipped for Windows:

```
builder.ConfigureLifecycleEvents(events =>
		{
#if WINDOWS10_0_17763_0_OR_GREATER
                events.AddWindows(wndLifeCycleBuilder =>
                {
                    wndLifeCycleBuilder.OnWindowCreated(window =>
                    {
                        window.SystemBackdrop = new MicaBackdrop { Kind = MicaKind.Base };

                        var titleBar = window.GetAppWindow()!.TitleBar;

                        titleBar.PreferredHeightOption = TitleBarHeightOption.Tall;

                        window.ExtendsContentIntoTitleBar = false;

                        IntPtr nativeWindowHandle = WinRT.Interop.WindowNative.GetWindowHandle(window);
                        WindowId win32WindowsId = Win32Interop.GetWindowIdFromWindow(nativeWindowHandle);
                        AppWindow winuiAppWindow = AppWindow.GetFromWindowId(win32WindowsId);

                        if (winuiAppWindow.Presenter is OverlappedPresenter p)
                        {
                            p.SetBorderAndTitleBar(true, true);
                        }
                    });
                });
#endif
		});
```
